openlibrary/plugins/openlibrary/js/ol.js

```diff
@@ -72,13 +72,6 @@ $(window).scroll(function(){
   if(isScrolledIntoView(scroller)){$("#scrollBtm").show();}else{$("#scrollBtm").hide();}
```

Comment:
no action required on this PR, request for feedback/thoughts:
Should our general policy be to unfurl such methods onto multiple lines?